src/univariate/continuous/normal.jl

```diff
@@ -68,9 +68,157 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
 
-#### Evaluation
+#### Evaluation (see JuliaStats/Distributions.jl/issues/708)
+
+xval(μ::Real, σ::Real, z::Number) = μ + σ * z
+zval(μ::Real, σ::Real, x::Number) = (x - μ) / σ
+
+# pdf
+normpdf(z::Real) = exp(-abs2(z)/2) * invsqrt2π
```

Comment:
I don't think we should implement the `normX` versions.

Reply:
I think we should just define these for the `Distributions` API, i.e. this method would simply be `pdf(::Normal, z)`

Reply:
It could be useful to have a separate "basic" functions. i.e. `normpdf(z)`, but we don't need `normpdf(x,mu,sigma)` versions.

similarly, `normcdf`, but you don't need `normccdf` since this is trivial.